### PR TITLE
Better German Translation

### PR DESCRIPTION
**Pull request by MinIsMin**
Turning the language to German breaks most text in detective and traitor shops.
I have inserted better line breaks to prevent this and updated some text.

My changes are listed in the 'Modified text pieces' section.

### CODE REVIEW

**Comment by Kefta** [comment]
Could you replace the translations at the locations they were before? The date comments are for new strings added to updates, not for modifying previous ones.

**Comment by MinIsMin** [comment]
Ah, ok. I misunderstood 'New and modified pieces of text'.
Done.

**Comment by svdm** [comment]
Nice, thanks!

**Comment by MinIsMin** [comment]
I'm sorry but I cut a character away.

https://github.com/MinIsMin/garrysmod/commit/dd510247e4727c146e6cd4248960f352c44df180
Line 456: 'einige' instead of 'inige'

**Comment by svdm** [comment]
No problem, I changed it myself just now in 9b9e7d9f03fd7fd9

File: garrysmod/gamemodes/terrortown/gamemode/lang/german.lua [hunk 1]
```diff
@@ -1047,3 +999,60 @@ L.set_cross_size = "Fadenkreuz Größe"
...
+ nutzt normale Pistolenmunition.
+ 
+ Opfer schreien nicht, wenn sie damit getötet werden.]]
+ 
+L.binoc_desc      = [[
+ Zoom an eine Leiche ran um sie aus einer
+ großen Entfernung zu identifizieren.
+ 
+ Unbegrenzt nutzbar, aber das Identifizieren dauert
+ inige Sekunden.]]
```
Comment (doctorluk) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: I guess an "e" got stripped. It should be _einige Sekunden_ and not _inige Sekunden_.

**Comment by doctorluk** [comment]
Whoopsie. Looks like this got catched already. Sorry, didn't scroll all the way through the comments!

**Pull request merged**